core/src/services/cos/backend.rs

```diff
@@ -299,9 +299,12 @@ impl Accessor for CosBackend {
     }
 
     async fn create_dir(&self, path: &str, _: OpCreateDir) -> Result<RpCreateDir> {
-        let mut req =
-            self.core
-                .cos_put_object_request(path, Some(0), None, None, None, AsyncBody::Empty)?;
+        let mut req = self.core.cos_put_object_request(
+            path,
+            Some(0),
+            &Default::default(),
```

Comment:
Use `&OpWrite::default()` to make reader happy.

Reply:
Yeah, That's what I am thinking...